### ISSUE TITLE
headComment not working as expected

### ISSUE THREAD

**Issue by MisterMX** [mikefarah/yq#1000]
**Describe the bug**

When I run the sample for the `headComment` operator [as provided in the docs](https://mikefarah.gitbook.io/yq/operators/comment-operators#get-head-comment). The output does not work as expected.

Version of yq: 4.14.1
Operating system: linux
Installed via: binary release

**Input Yaml**
```yaml
# welcome!

a: cat # meow

# have a great day
```

**Command**
```bash
yq eval '. | headComment' sample.yml
```

**Actual output**
```
$yqLeadingContent$
welcome!

```

**Expected output**
```
welcome!
```

**Comment by mikefarah** [comment]
Yep that's a bug :/ will fix in the next release.

**Comment by mikefarah** [comment]
Fixed in 4.14.2 :)
